_config/debugbar.yml

```diff
@@ -51,3 +51,5 @@ SilverStripe\Admin\LeftAndMain:
 SilverStripe\Control\Controller:
   extensions:
     - LeKoala\DebugBar\Extension\ControllerExtension
+LeKoala\DebugBar\DebugBar:
+  show_namespaces: false
```

Comment:
Can you at least document the change? We've got a list in docs/en/configuration.md and a comment would be good too